Backend/app/dashboard/CommonAPIs/src/Dashboard/Common/Issue.hs

```diff
@@ -22,13 +22,34 @@ import Servant hiding (Summary)
 
 -- we need to save endpoint transactions only for POST, PUT, DELETE APIs
 data IssueEndpoint
-  = IssueListEndpoint
+  = IssueCategoryListEndpoint
+  | IssueListEndpoint
+  | IssueInfoEndpoint
```

Comment:
this datatype used for storing transaction in DB and not required for GET APIs. So it should be:

```
data IssueEndpoint
  = IssueUpdateEndpoint
  | IssueAddCommentEndpoint
```